rotkehlchen/globaldb/handler.py

```diff
@@ -871,33 +871,37 @@ def get_token_detection_data(
             chain_id: ChainID,
             exceptions: set[ChecksumEvmAddress],
             protocol: str | None = None,
-    ) -> list[EvmTokenDetectionData]:
+    ) -> tuple[list[EvmTokenDetectionData], list[EvmTokenDetectionData]]:
```

Comment:
Please whenever you make such changes explain in the docstring what the function is doing. You need to update it.